marshal/admin.go

```diff
@@ -155,35 +155,57 @@ func (a *consumerGroupAdmin) releaseClaims(resetOffset bool) error {
 	}
 }
 
-// heartbeatLoop hearbeats as if we had a claim to this partition and were simply
+// Send a single heartbeat using the given offset.
+func (a *consumerGroupAdmin) heartbeat(
+	topic string, partID int, offset int64) (ok bool) {
```

Comment:
nit: No named return variables please, and these two lines should be one. They should be within 100 character line length limit.